data/patch.json

```diff
@@ -815,6 +815,10 @@
         "system-family-name": {
             "syntax": "caption | icon | menu | message-box | small-caption | status-bar",
             "comment": "new definition on font-4, https://drafts.csswg.org/css-fonts-4/#system-family-name-value"
+        },
+        "parentheses-block": {
+            "comment": "missed, https://drafts.csswg.org/css-syntax/#typedef-parentheses-block",
```

Comment:
It looks like there is no anchor named `#typedef-parentheses-block` in the specs page?